### PR TITLE
README: improve overall quality

### PR DESCRIPTION
**Pull request by trueNAHO**
- README: add empty lines for consistent paragraph formatting
- README: capitalize bullet points consistently
- README: punctuate bullet points consistently
- README: format code blocks with `bash` instead of `shell`
    - The GitHub previewer has better color support for `bash` code blocks than for `shell` code blocks`.
- README: remove initial code block indentation
- README: code format commands and command names

### CODE REVIEW

**Comment by newren** [comment]
The commits generally look good.  I like the split into separate commits, and you describe each change well.

While changing "filter-branch" to "`filter-branch`" is generally good and correct, changing "git filter-branch" to "git `filter-branch`" is not correct; the backtick should start before "git" in that case.  I caught at least one example of that problem.

However, there's one big problem here -- I need to you to use your real name and real email in the author & committer, and include a Signed-off-by line (https://github.com/git/git/blob/master/Documentation/SubmittingPatches#L351).  Can you do that?  If so (and if you fix the backtick issue I mentioned above), I'd be happy to merge the changes.

**Review by DEMO861 — APPROVED** [review]

**Comment by newren** [comment]
No response to the request to add the signed-off-by.  It's a shame to lose these changes, but I can't merge without that.

**Pull request closed without merging**
